modules/tensor_mechanics/test/tests/torque_reaction/tests

```diff
@@ -1,34 +1,53 @@
 [Tests]
   [./torque_reaction_2D]
+    design = TorqueReaction.md
+    issues = '#6484'
+    requirement = 'The TorqueReaction Postprocessor shall compute the torque for an applied deformation at the end of a beam in 2D'
```

Comment:
Are these supposed to all start with 'The system shall.....'? I left my SQA notes in the office

Reply:
Yeah, I wasn't sure. I think you're right.